aries-core/src/js/designer/index.js

```diff
@@ -0,0 +1,16 @@
+import { Blank } from 'grommet-icons';
+import { components } from './components';
+
+const structure = [
+  {
+    name: 'Aries',
+    Icon: Blank,
+    components: ['Tile'],
```

Comment:
I might be missing something, but what about the other components? 
Or are you intentionally just using one Tile and an Icon for the structure?
I guess I'm not sure what the `structure` means.

Reply:
I haven't been keeping this up to date, since this pull request is so old ;)
I thought you could check it for general issues and then I'll add the other components.

Reply:
The general direction of this PR LGTM.